recipes/openfst/all/patches/1.8.2-allocate-hint.patch

```diff
@@ -0,0 +1,34 @@
+--- src/include/fst/memory.h
++++ src/include/fst/memory.h
+@@ -237,11 +237,11 @@
+   explicit BlockAllocator(const BlockAllocator<U> &arena_alloc)
+       : arenas_(arena_alloc.Arenas()) {}
+ 
+-  T *allocate(size_type n, const void *hint = nullptr) {
++  T *allocate(size_type n) {
```

Comment:
Is this patch listed in the upstream? Usually only accept patches related to build configuration or security patch. We avoid feature/hotfix which is related to the source code.

Reply:
allocate with hint is never used in this code, and it breaks compatibility with c++20. Do u have any suggestions to deal with such case?

Reply:
openfst does not provide a channel for community feedback or PR. So it's nearly impossible for such a patch. Since the patch has no effects, I would like to recommend option 4